client/src/components/pokemons/Pokemon.js

```diff
@@ -133,10 +141,19 @@ const Pokemon = props => {
                         {type.type.name}
                       </span>
                     ))}
-                </h5>
-                <h5>Base Experience: {base_experience}</h5>
-                <h5>Weight: {weight}</h5>
-                <h5>Height: {height}</h5>
+                </h6>
+                <h6>Base Experience: {base_experience}</h6>
+                <h6>Weight: {weight}</h6>
+                <h6>Height: {height}</h6>
+                {level && <h6>Level: {level}</h6>}
+                {moves && (
```

Comment:
this is a bit tricky -> try hardcoding `moves` to be `[]` here 😄

`[] && 'lol'` evaluates to `'lol'`, because empty array is a "truthy" value, so you will display an empty Moves list -> it's safer to call `moves && moves.length > 0 && (` or use a library which does that for you

To read: https://medium.com/coding-in-simple-english/what-are-truthy-values-in-javascript-e037bdfa76f8